### PR TITLE
DMTCP-2.3.1

### PR DESCRIPTION
**Pull request by jordiblasco**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+toolchain = {'name': 'gmpolf', 'version': '1.4.8'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ["http://downloads.sourceforge.net/project/dmtcp/dmtcp-2.x/%s/" % (version)]
+
+configopts = "--enable-timing --enable-infiniband-support " 
+
+sanity_check_paths = {
+                      'files': ['bin/dmtcp_checkpoint', 'bin/dmtcp_command', 'bin/dmtcp_coordinator' , 'bin/dmtcp_discover_rm' , 'bin/dmtcp_launch' , 'bin/dmtcp_restart' ],
+                      'dirs': []
```
Comment (boegel) [inline easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb]: fix indent please

File: easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+toolchain = {'name': 'gmpolf', 'version': '1.4.8'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ["http://downloads.sourceforge.net/project/dmtcp/dmtcp-2.x/%s/" % (version)]
+
+configopts = "--enable-timing --enable-infiniband-support " 
+
+sanity_check_paths = {
+                      'files': ['bin/dmtcp_checkpoint', 'bin/dmtcp_command', 'bin/dmtcp_coordinator' , 'bin/dmtcp_discover_rm' , 'bin/dmtcp_launch' , 'bin/dmtcp_restart' ],
```
Comment (boegel) [inline easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb]: avoid long line, wrap if longer than 120 chars

File: easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+name = 'DMTCP'
+version = '2.3.1'
+
+homepage = 'http://dmtcp.sourceforge.net//index.html'
+description = """DMTCP (Distributed MultiThreaded Checkpointing) is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications."""
+
+toolchain = {'name': 'gmpolf', 'version': '1.4.8'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ["http://downloads.sourceforge.net/project/dmtcp/dmtcp-2.x/%s/" % (version)]
```
Comment (boegel) [inline easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb]: use `%(version)s` template

File: easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Authors::   Jordi Blasco <jordi.blasco@nesi.org.nz>
+#
+##
+
+name = 'DMTCP'
+version = '2.3.1'
+
+homepage = 'http://dmtcp.sourceforge.net//index.html'
+description = """DMTCP (Distributed MultiThreaded Checkpointing) is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications."""
```
Comment (boegel) [inline easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-gmpolf-1.4.8.eb]: break up long line into multiple lines, start additional lines with a space

File: easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-goolf-1.5.14.eb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+name = 'DMTCP'
+version = '2.3.1'
+
+homepage = 'http://dmtcp.sourceforge.net//index.html'
+description = """DMTCP (Distributed MultiThreaded Checkpointing) is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ["http://downloads.sourceforge.net/project/dmtcp/dmtcp-2.x/%s/" % (version)]
```
Comment (boegel) [inline easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-goolf-1.5.14.eb]: `%(version)s`

File: easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-goolf-1.5.14.eb [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Authors::   Jordi Blasco <jordi.blasco@nesi.org.nz>
+#
+##
+
+name = 'DMTCP'
+version = '2.3.1'
+
+homepage = 'http://dmtcp.sourceforge.net//index.html'
+description = """DMTCP (Distributed MultiThreaded Checkpointing) is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications."""
```
Comment (boegel) [inline easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-goolf-1.5.14.eb]: break long line

File: easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-goolf-1.5.14.eb [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ["http://downloads.sourceforge.net/project/dmtcp/dmtcp-2.x/%s/" % (version)]
+
+configopts = "--enable-timing --enable-infiniband-support OPENMPI_MPICC=$(which mpicc) OPENMPI_MPIRUN=$(which mpirun) " # Apparently, only OpenMPI requires these extra options.
+
+sanity_check_paths = {
+                      'files': ['bin/dmtcp_checkpoint', 'bin/dmtcp_command', 'bin/dmtcp_coordinator' , 'bin/dmtcp_discover_rm' , 'bin/dmtcp_launch' , 'bin/dmtcp_restart' ],
```
Comment (boegel) [inline easybuild/easyconfigs/d/DMTCP/DMTCP-2.3.1-goolf-1.5.14.eb]: fix indent, avoid long line

**Comment by migueldiascosta** [comment]
@jordiblasco, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**